liquidator/test/Liquidator.js

```diff
@@ -105,11 +106,12 @@ contract("Liquidator.js", function(accounts) {
     await syntheticToken.approve(emp.address, toWei("100000000"), { from: sponsor3 });
     await syntheticToken.approve(emp.address, toWei("100000000"), { from: liquidatorBot });
 
-    // Create a new instance of the ExpiringMultiPartyClient to construct the liquidator
+    // Create a new instance of the ExpiringMultiPartyClient & gasEstimator to construct the liquidator
     empClient = new ExpiringMultiPartyClient(ExpiringMultiParty.abi, web3, emp.address);
+    gasEstimator = new GasEstimator();
```

Comment:
nit: add `let gasEstimator` to line 29

Reply:
as with the `empClient` this actually is not defined anywhere and it does not cause any problems. If I declare an instance of `gasEstimator` up above then we should do the same with the other objects we create.